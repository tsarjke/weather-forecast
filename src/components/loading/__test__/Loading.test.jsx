import { render, screen } from '@testing-library/react';
import Loading from '../Loading';

describe('Loading', () => {
  test('should render this component without any mistakes', () => {
    render(<Loading />);

    const loadingElem = screen.getByTestId('loading');
    expect(loadingElem).toMatchSnapshot();
  });
});
