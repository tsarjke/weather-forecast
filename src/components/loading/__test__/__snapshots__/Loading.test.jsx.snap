// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render this component without any mistakes 1`] = `
<div
  class="loading"
  data-testid="loading"
>
  <svg
    height="120"
    id="clackers"
    width="300"
  >
    <svg>
      <path
        d="M 90 90 A 90 90 0 0 1 0 0"
        fill="none"
        id="arc-left-up"
      />
    </svg>
    <svg>
      <path
        d="M 100 90 A 90 90 0 0 0 190 0"
        fill="none"
        id="arc-right-up"
      />
    </svg>
    <text
      fill="#0e25ee"
      font-family="Helvetica Neue,Helvetica,Arial"
      font-size="22"
      text-anchor="middle"
      x="150"
      y="50"
    >
      L O A D I N G
    </text>
    <circle
      cx="15"
      cy="15"
      r="15"
    >
      <animatemotion
        calcMode="linear"
        dur="1.5s"
        keyPoints="0.0;0.19;0.36;0.51;0.64;0.75;0.84;0.91;0.96;0.99;1.0;0.99;0.96;0.91;0.84;0.75;0.64;0.51;0.36;0.19;0.0;0.0;0.05;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0"
        keyTimes="0.0;0.025;0.05;0.075;0.1;0.125;0.15;0.175;0.2;0.225;0.25;0.275;0.3;0.325;0.35;0.375;0.4;0.425;0.45;0.475;0.5;0.525;0.55;0.575;0.6;0.625;0.65;0.675;0.7;0.725;0.75;0.775;0.8;0.825;0.85;0.875;0.9;0.925;0.95;0.975;1.0"
        repeatCount="indefinite"
      >
        <mpath
          xlink:href="#arc-left-up"
        />
      </animatemotion>
    </circle>
    <circle
      cx="135"
      cy="105"
      r="15"
    />
    <circle
      cx="165"
      cy="105"
      r="15"
    />
    <circle
      cx="95"
      cy="15"
      r="15"
    >
      <animatemotion
        calcMode="linear"
        dur="1.5s"
        keyPoints="0.0;0.0;0.05;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0;0.0;0.19;0.36;0.51;0.64;0.75;0.84;0.91;0.96;0.99;1.0;0.99;0.96;0.91;0.84;0.75;0.64;0.51;0.36;0.19;0.0"
        keyTimes="0.0;0.025;0.05;0.075;0.1;0.125;0.15;0.175;0.2;0.225;0.25;0.275;0.3;0.325;0.35;0.375;0.4;0.425;0.45;0.475;0.5;0.525;0.55;0.575;0.6;0.625;0.65;0.675;0.7;0.725;0.75;0.775;0.8;0.825;0.85;0.875;0.9;0.925;0.95;0.975;1.0"
        repeatCount="indefinite"
      >
        <mpath
          xlink:href="#arc-right-up"
        />
      </animatemotion>
    </circle>
  </svg>
</div>
`;
